SConstruct

```diff
@@ -18,15 +18,30 @@ AddOption('--asan',
 cython_dependencies = [Value(v) for v in (sys.version, distutils.__version__, Cython.__version__)]
 Export('cython_dependencies')
 
+Help("\nType: 'scons cc=1' for cross-compilation.\n")
```

Comment:
Maybe not use `cc` as a variable/flag, that could be confused with the `CC` build variable specifying the C compiler to use.